README.md

```diff
@@ -7,6 +7,8 @@
 
 If you have a question, or need assistance during class, **please create an issue and mention your teacher!**.
 
+- [check out our website]  (http://www.voltaix.com/)
```

Comment:
@donnabenesh if you remove the space between `website] (http:` it will be displayed as a link.